---
title: 'Widespread macromolecular interaction perturbations in human genetic disorders'

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here
# and it will be replaced with their full name and linked to their profile.
authors:
  - Sahni, Nidhi
  - Yi, Song
  - Taipale, Mikko
  - Bass, Juan I Fuxman
  - Coulombe-Huntington, Jasmin
  - Yang, Fan
  - Peng, Jian
  - Weile, Jochen
  - Karras, Georgios I 
  - Wang, Yang
  - Kovacs, Istvan A
  - Atanas Kamburov
  - Irina Krykbaeva
  - Mandy H. Lam
  - George Tucker
  - Vikram Khurana
  - Amitabh Sharma
  - Yang-Yu Liu
  - Nozomu Yachie
  - Quan Zhong
  - Yun Shen
  - Alexandre Palagi
  - Adriana San-Miguel
  - Changyu Fan
  - Dawit Balcha
  - Amelie Dricot
  - Daniel M. Jordan
  - Jennifer M. Walsh
  - Akash A. Shah
  - Xinping Yang
  - Ani K. Stoyanova
  - Alex Leighton
  - Michael A. Calderwood
  - Yves Jacob
  - Michael E. Cusick
  - Kourosh Salehi-Ashtiani
  - Luke J. Whitesell
  - Shamil Sunyaev,12,16 Bonnie Berger
  - Albert-La´ szlo´ Baraba´ si
  - Benoit Charloteaux
  - David E. Hill
  - Tong Hao
  - Frederick P. Roth

# Author notes (optional)

date: '2015-04-23'
doi: ''

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ['2']

# Publication name and optional abbreviated publication name.
publication: Cell
publication_short:

abstract: The three-dimensional conformation of a genome can be profiled using Hi-C, a technique that combines chromatin conformation capture with high-throughput sequencing. However, structural variations often yield features that can be mistaken for chromosomal interactions. Here, we describe a computational method HiNT (Hi-C for copy Number variation and Translocation detection), which detects copy number variations and interchromosomal translocations within Hi-C data with breakpoints at single base-pair resolution. We demonstrate that HiNT outperforms existing methods on both simulated and real data. We also show that Hi-C can supplement whole-genome sequencing in structure variant detection by locating breakpoints in repetitive regions.

# Summary. An optional shortened abstract.

tags: []

# Display this page in the Featured widget?
featured: true

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: 'https://www.cell.com/cell/fulltext/S0092-8674(15)00430-4'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# image:
 #  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/pLCdAaMFLTE)'
 #  focal_point: ''
 #  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
